packages/p2p/src/components/page-return/page-return.tsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import classNames from 'classnames';
 import { Icon, Text } from '@deriv/components';
-import PropTypes from 'prop-types';
-import './page-return.scss';
 
-const PageReturn = ({ className, onClick, page_title }) => {
+type PageReturnProps = {
```

Comment:
```suggestion
type TPageReturn = {
```

Naming convention for our types: `TComponentName`